recipes/libcurl/all/conanfile.py

```diff
@@ -5,7 +5,10 @@
 from conans import ConanFile, AutoToolsBuildEnvironment, CMake, tools
 from conans.errors import ConanInvalidConfiguration
 from conan.tools.microsoft import is_msvc
-from conan.tools.build import cross_building
+try:
+    from conan.tools.cross_building import cross_building
+except ImportError:
+    from conan.tools.build.cross_building import cross_building
```

Comment:
```suggestion
from conan.tools.build.cross_building import cross_building
```
We need to move forward to Conan 2.0